src/App.tsx

```diff
@@ -1,24 +1,244 @@
-/* eslint-disable max-len */
```

Comment:
Have you added an additional library you've installed to `package.json` dependency list?
![image](https://user-images.githubusercontent.com/86923089/220184600-8232b829-ec02-4a0d-8ac4-491c7b323aeb.png)

If not - please, do it